[2024-03-05T20:54:31.578+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:31.583+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:31.583+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T20:54:31.590+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-05T20:54:31.592+0000] {standard_task_runner.py:60} INFO - Started process 299 to run task
[2024-03-05T20:54:31.594+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmplhh_gsno']
[2024-03-05T20:54:31.595+0000] {standard_task_runner.py:88} INFO - Job 25: Subtask download_dataset
[2024-03-05T20:54:31.619+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T20:54:31.644+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-03-05T20:54:31.660+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-03-05T20:54:31.661+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-05T20:54:31.661+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh 2023 10']
[2024-03-05T20:54:31.666+0000] {subprocess.py:86} INFO - Output:
[2024-03-05T20:54:31.668+0000] {subprocess.py:93} INFO - 
[2024-03-05T20:54:31.668+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/10
[2024-03-05T20:54:31.668+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/10
[2024-03-05T20:54:31.669+0000] {subprocess.py:93} INFO - /opt/***/include/download_bike_dataset.sh: 14: wget: Permission denied
[2024-03-05T20:54:31.669+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-03-05T20:54:31.673+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-05T20:54:31.676+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240305T205431, end_date=20240305T205431
[2024-03-05T20:54:31.682+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 25 for task download_dataset (Bash command failed. The command returned a non-zero exit code 1.; 299)
[2024-03-05T20:54:31.686+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T20:54:31.697+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-05T21:13:01.846+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-05T21:13:01.851+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-05T21:13:01.852+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T21:13:01.858+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-05T21:13:01.861+0000] {standard_task_runner.py:60} INFO - Started process 813 to run task
[2024-03-05T21:13:01.863+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmp6vz2t0hi']
[2024-03-05T21:13:01.865+0000] {standard_task_runner.py:88} INFO - Job 55: Subtask download_dataset
[2024-03-05T21:13:01.890+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T21:13:01.913+0000] {abstractoperator.py:707} ERROR - Exception rendering Jinja template for task 'download_dataset', field 'bash_command'. Template: "sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh {{ ds.strftime('%Y') }} {{ ds.strftime('%m') }}"
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:13:01.914+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2465, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2877, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1241, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:13:01.917+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240305T211301, end_date=20240305T211301
[2024-03-05T21:13:01.923+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 55 for task download_dataset ('str object' has no attribute 'strftime'; 813)
[2024-03-05T21:13:01.955+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T21:13:01.968+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T13:27:32.118+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:32.124+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:32.124+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T13:27:32.131+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-06T13:27:32.133+0000] {standard_task_runner.py:60} INFO - Started process 544 to run task
[2024-03-06T13:27:32.135+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpcm0nn3bt']
[2024-03-06T13:27:32.136+0000] {standard_task_runner.py:88} INFO - Job 84: Subtask download_dataset
[2024-03-06T13:27:32.160+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host 45431f0a53e1
[2024-03-06T13:27:32.190+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T13:27:32.207+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-03-06T13:27:32.207+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T13:27:32.208+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 11']
[2024-03-06T13:27:32.213+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T13:27:32.214+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:32.214+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/11
[2024-03-06T13:27:32.218+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T13:27:32.219+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T13:27:33.999+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 14 2699k   14  380k    0     0   318k      0  0:00:08  0:00:01  0:00:07  318k100 2699k  100 2699k    0     0  1516k      0  0:00:01  0:00:01 --:--:-- 1515k
[2024-03-06T13:27:34.002+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:34.002+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T13:27:34.003+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T13:27:34.019+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240306T132732, end_date=20240306T132734
[2024-03-06T13:27:34.034+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T13:27:34.043+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T17:40:17.276+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-06T17:40:17.281+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-06T17:40:17.281+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T17:40:17.288+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-06T17:40:17.292+0000] {standard_task_runner.py:60} INFO - Started process 1463 to run task
[2024-03-06T17:40:17.294+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmp8rb5ln5t']
[2024-03-06T17:40:17.295+0000] {standard_task_runner.py:88} INFO - Job 119: Subtask download_dataset
[2024-03-06T17:40:17.319+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host e43ee8560afa
[2024-03-06T17:40:17.350+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T17:40:17.367+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-03-06T17:40:17.368+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T17:40:17.369+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 11']
[2024-03-06T17:40:17.374+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T17:40:17.376+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:40:17.376+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/11
[2024-03-06T17:40:17.380+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T17:40:17.381+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T17:40:19.074+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 2699k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 28 2699k   28  781k    0     0   521k      0  0:00:05  0:00:01  0:00:04  520k100 2699k  100 2699k    0     0  1593k      0  0:00:01  0:00:01 --:--:-- 1593k
[2024-03-06T17:40:19.078+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv.zip
[2024-03-06T17:40:19.135+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv
[2024-03-06T17:40:19.136+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/__MACOSX/._JC-202311-citibike-tripdata.csv
[2024-03-06T17:40:19.384+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv (deflated 82%)
[2024-03-06T17:40:19.386+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:40:19.386+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T17:40:19.386+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T17:40:19.400+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240306T174017, end_date=20240306T174019
[2024-03-06T17:40:19.433+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T17:40:19.444+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-07T14:09:11.657+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-07T14:09:11.663+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-07T14:09:11.663+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T14:09:11.670+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-07T14:09:11.674+0000] {standard_task_runner.py:60} INFO - Started process 1525 to run task
[2024-03-07T14:09:11.675+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpaz13qtck']
[2024-03-07T14:09:11.677+0000] {standard_task_runner.py:88} INFO - Job 174: Subtask download_dataset
[2024-03-07T14:09:11.701+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host 0430cbdaac75
[2024-03-07T14:09:11.726+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T14:09:11.837+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-03-07T14:09:11.838+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T14:09:11.838+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 11']
[2024-03-07T14:09:11.845+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T14:09:11.846+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:09:11.847+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/11
[2024-03-07T14:09:11.851+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T14:09:11.851+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T14:09:13.889+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 19 2699k   19  525k    0     0   331k      0  0:00:08  0:00:01  0:00:07  331k100 2699k  100 2699k    0     0  1324k      0  0:00:02  0:00:02 --:--:-- 1325k
[2024-03-07T14:09:13.891+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv.zip
[2024-03-07T14:09:13.949+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv
[2024-03-07T14:09:13.949+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/__MACOSX/._JC-202311-citibike-tripdata.csv
[2024-03-07T14:09:14.197+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv (deflated 82%)
[2024-03-07T14:09:14.199+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:09:14.199+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T14:09:14.199+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv.zip
[2024-03-07T14:09:14.199+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T14:09:14.213+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240307T140911, end_date=20240307T140914
[2024-03-07T14:09:14.258+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T14:09:14.273+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-07T17:56:56.432+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-07T17:56:56.437+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-03-07T17:56:56.437+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T17:56:56.444+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-10-01 00:00:00+00:00
[2024-03-07T17:56:56.449+0000] {standard_task_runner.py:60} INFO - Started process 640 to run task
[2024-03-07T17:56:56.451+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmptyz6xhd3']
[2024-03-07T17:56:56.452+0000] {standard_task_runner.py:88} INFO - Job 235: Subtask download_dataset
[2024-03-07T17:56:56.478+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-10-01T00:00:00+00:00 [running]> on host a0c898e5643f
[2024-03-07T17:56:56.507+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T17:56:56.526+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-03-07T17:56:56.527+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T17:56:56.528+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 11']
[2024-03-07T17:56:56.535+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T17:56:56.537+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:56:56.537+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/11
[2024-03-07T17:56:56.541+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T17:56:56.541+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T17:56:58.671+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 2699k    0  8486    0     0   8540      0  0:05:23 --:--:--  0:05:23  8537 30 2699k   30  823k    0     0   461k      0  0:00:05  0:00:01  0:00:04  460k100 2699k  100 2699k    0     0  1267k      0  0:00:02  0:00:02 --:--:-- 1267k
[2024-03-07T17:56:58.675+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv.zip
[2024-03-07T17:56:58.733+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv
[2024-03-07T17:56:58.734+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/11/__MACOSX/._JC-202311-citibike-tripdata.csv
[2024-03-07T17:56:58.980+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv (deflated 82%)
[2024-03-07T17:56:58.981+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:56:58.981+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T17:56:58.981+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/11/JC-202311-citibike-tripdata.csv.zip
[2024-03-07T17:56:58.982+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T17:56:58.996+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231001T000000, start_date=20240307T175656, end_date=20240307T175658
[2024-03-07T17:56:59.031+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T17:56:59.045+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
